### PR TITLE
Add new Other Apps dropdown

### LINKED ISSUE #75: Small area where other pages links can be clicked when hidden
Just an issue of pointer-events: none; not happening. will fix this. just putting this here to remind myself

https://user-images.githubusercontent.com/33783503/204943859-a2d02e53-2f68-4c91-a4e6-e5884f08c891.mp4

### PR DESCRIPTION
**Pull request by Tanza3D**
Adds new Other Apps dropdown on Desktop and Mobile
image
image
Closes #75 #24

### CODE REVIEW

**Review by minusQuantumNeko — APPROVED** [review]
gud

File: global/php/navbar.php [hunk 1]
```diff
@@ -682,4 +729,4 @@
 <script type="text/javascript" src="<?= ROOT_URL; ?>/global/js/variables.js?v=<?= OSEKAI_VERSION; ?>"></script>
 <script type="text/javascript" src="<?= ROOT_URL; ?>/global/js/main.js?v=<?= OSEKAI_VERSION; ?>"></script>
 
-<script src="<?= ROOT_URL; ?>/global/js/navbar.js"></script>
+<script src="<?= ROOT_URL; ?>/global/js/navbar.js"></script>
```
Comment (EXtremeExploit) [inline global/php/navbar.php:732]:
```suggestion
<script src="<?= ROOT_URL; ?>/global/js/navbar.js"></script>

```

File: global/php/navbar.php [hunk 2]
```diff
@@ -198,12 +237,20 @@
...
-                    <a class="osekai__apps-dropdown-other-content-button" href="/misc/translators/">
-                        Translators
-                    </a>
+                    <?php
+                    foreach ($otherApps as $oapp) {
+                        // would name this $app but for whatever fucking reason php decides that it's
+                        // a great idea to replace the variable "$app" with it and never put it 
+                        // back globally across the entire project if you dare to do that so we
+                        // have to kinda just be careful not to?? i hate this
+                        echo "<a class=\"osekai__apps-dropdown-other-content-button\" href=\"{$oapp['link']}\">
```
Comment (EXtremeExploit) [inline global/php/navbar.php:249]: imo would be better to use single quote marks here so the escaping of " is not needed, just a bit more readable

File: global/php/navbar.php [hunk 3]
```diff
@@ -135,60 +150,84 @@
...
+            </a>
+        </div>
+        <div class="osekai__apps-dropdown-mobile-section" style="--height: 70px;">
+            <?php
+            foreach ($otherApps as $oapp) {
+                // would name this $app but for whatever fucking reason php decides that it's
+                // a great idea to replace the variable "$app" with it and never put it 
+                // back globally across the entire project if you dare to do that so we
+                // have to kinda just be careful not to?? i hate this
+                echo "<a class=\"osekai__apps-dropdown-mobile-button\" href=\"{$oapp['link']}\">
```
Comment (EXtremeExploit) [inline global/php/navbar.php:221]: same here

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: global/php/navbar.php (hunk 2 above)
Comment (Tanza3D) [reply global/php/navbar.php:249]: last i checked those don't work in html?

Comment (EXtremeExploit) [reply global/php/navbar.php:249]: i mean the ones the echo is using, not the ones that are given for the class or href

Comment (Tanza3D) [reply global/php/navbar.php:249]: then you can't use {$variable}, that only works with double quotes

Comment (EXtremeExploit) [reply global/php/navbar.php:249]: my bad then, im still kinda new to php stuff

**Review by EXtremeExploit — APPROVED** [review]

**Pull request merged**